src/MountedTraversal.js

```diff
@@ -20,10 +20,11 @@ import {
   isElement,
   findDOMNode,
 } from './react-compat';
-import { REACT013, REACT014 } from './version';
+import { REACT013 } from './version';
 
 export function internalInstance(inst) {
-  return inst._reactInternalInstance;
+  return inst._reactInternalInstance ||
+    inst[Object.keys(inst).find(key => key.match(/^__reactInternalInstance\$/))];
```

Comment:
`.find` is ES6 - you'll need `.indexOf` or `.filter` here (and throughout) to keep it ES5 - or else use http://npmjs.com/array.prototype.find :-)